### PR TITLE
New user/add hooroo

### PR DESCRIPTION
**Pull request by tancnle**
Love your work, team!!!

_Added bonus_: Remove executable permission on one of existing user image file.

### CODE REVIEW

File: images/users/hooroo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...4.636,50.928 61.428,52.059 61.428,61.113 C61.428,63.699 60.659,67.014 60.659,67.014 C59.933,70.173 58.743,75.345 58.027,78.514 L49.574,115.262 C48.854,118.425 50.92,121.013 54.169,121.013 L67.277,121.013 C70.524,121.013 73.777,118.425 74.517,115.262 L80.52,89.722 C81.529,100.991 88.323,117.569 111.531,117.569 C124.016,117.569 133.696,112.197 140.167,104.487 C146.189,111.916 155.872,117.569 171.163,117.569 C202.092,117.569 219.719,91.928 219.719,68.369 M129.677,78.89 C129.677,87.4 125.422,102.292 112.654,102.292 C103.144,102.292 99.885,94.658 99.885,87.4 C99.885,77.765 105.268,64.615 116.538,64.118 C121.917,63.864 125.679,66.749 127.548,69.628 C128.803,71.631 129.677,74.506 129.677,78.89 M194.397,68.05 C194.397,78.946 188.949,98.014 172.602,98.014 C160.424,98.014 156.255,88.245 156.255,78.946 C156.255,66.603 163.146,49.78 177.571,49.142 C184.46,48.816 189.273,52.505 191.673,56.191 C193.276,58.76 194.397,62.441 194.397,68.05 M258.328,152.86 C281.221,167.118 308.992,171.911 335.419,169.037 C356.127,166.791 381.939,158.75 391.811,139.316 C394.832,133.365 391.008,114.977 387.06,118.568 C353.602,148.956 303.099,161.076 260.892,143.023 C257.782,141.691 254.4,150.42 258.328,152.86"/></svg>
```
Comment (aaronang) [inline images/users/hooroo.svg]: I looked at the SVG and for some reason there is lot of blank space on the right of the logo? Could you fix this?

**Comment by tancnle** [comment]
Hi @aaronang, I have adjusted the aspect ratio to remove those white spaces.

**Comment by existentialism** [comment]
@tancnle thanks for the PR!

**Comment by aaronang** [comment]
Wait, shoudn't the SVG have a viewbox of 500x200?

**Comment by existentialism** [comment]
@aaronang it's fine.

**Pull request merged**